src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1835,6 +1834,14 @@ protected override void BeginProcessing()
             catch (CommandNotFoundException)
             {
                 startInfo.FileName = FilePath;
+#if UNIX
+                //Arguments are passed incorrectly to the executable used for ShellExecute and not to filename https://github.com/dotnet/corefx/issues/30718
+                //so don't use ShellExecute if arguments are specified
+
+                //Linux relies on `xdg-open` and macOS relies on `open` which behave differently than Windows ShellExecute when running console commands
+                //as a new console will be opened.  So to avoid that, we only use ShellExecute on non-Windows if the filename is not an actual command (like a URI)
```

Comment:
Please add space after `//` in comments.

Reply:
I was following the existing comments

Reply:
Yes, I see. But if we continue use CodeFactor we have to fix every code block without following file style.

Reply:
I'll update this file with whitespace as a separate commit so it doesn't make the overall code review noisy